bakery/tasks/build-pdf.yml

```diff
@@ -0,0 +1,21 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: openstax/princexml
+
+inputs:
+  - name: book
+  - name: mathified-book
+outputs:
+  - name: artifacts
+
+run:
+  user: root
+  path: /bin/bash
+  args:
+    - -cxe
+    - |
+      book_dir="mathified-book/$(cat book/name)"
+      prince -v --output="artifacts/$(cat book/name).pdf" "$book_dir/collection.mathified.xhtml"
```

Comment:
can we add the accessibility flag?

Reply:
Do you know what it's called?

Reply:
@andymuddimer what's the accessibility flag you need?

Reply:
I was doing a quick investigation with @helenemccarron - she might have some input.. but this was the article I found with the attribute to add to PrinceXML command
- https://docs.google.com/document/d/1eHc6Y_JuiqnSdetH1A8bYuuB7ZL3w--35CIxVvK6ZSA

Reply:
@karenc the flag we tested is `--pdf-profile="PDF/UA-1"`

Reply:
It does produce errors though: 
```
prince  data/chemistry/collection.baked.xhtml -o mynewnewPDF.pdf --pdf-profile="PDF/UA-1"
prince: warning: possibly inappropriate use of L structure element on page 1390
prince: error: not identifying as PDF/UA-1 due to problems in structure tree```

Reply:
Without the CSS: ```prince: error: Formula structure element is missing alternative text on page 1919
prince: error: not identifying as PDF/UA-1 due to problems in structure tree```

Reply:
Hm... can we wait until it works before we add the flag?

Reply:
Definitively would be best, since this actually breaks the PDF generation. It looks like both the CSS and the markup need to be amended for this to work. I think we need to assess more thoroughly than my quick two-minute spike what would be needed before changing the CLI.